File: pgm/ABC_Dart/ABC_424_A.dart
import 'dart:collection';
import 'dart:io';
// import 'dart:math';

void main() {
  final [int a, int b, int c] = stdin
      .readLineSync()!
      .split(' ')
      .map(int.parse)
      .toList();
  const int two = 2;
  const String Yes = 'Yes';
  const String No = 'No';
  final Set<int> set = Set.unmodifiable(HashSet<int>.from([a, b, c]));
  if (set.length <= two) {
    print(Yes);
  } else {
    print(No);
  }
  exit(0);
}

File: pgm/ABC_Kotlin/ABC_424_A.kt
// import kotlin.math.*

const val two: Int = 2
const val Yes: String = "Yes"
const val No: String = "No"
const val space: String = " "

fun main() {
    val (a: Int, b: Int, c: Int) = readln().split(space).map { it.toInt() }
    val set: Set<Int> = hashSetOf(a, b, c)
    if (set.size <= two) {
        println(Yes)
    } else {
        println(No)
    }
    kotlin.system.exitProcess(0)
}
